aten/src/ATen/native/cudnn/Conv.cpp

```diff
@@ -268,6 +286,61 @@ struct ConvolutionArgs {
   }
 };
 
+std::string repro_from_args(const ConvolutionArgs& args) {
+  auto pybool = [](bool b) -> std::string { return b ? "True" : "False"; };
```

Comment:
Very minor, but you can just return a `const char*` here, it's all string literals